packages/react-next/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -39,30 +39,155 @@ import { IProcessedStyleSet } from '../../Styling';
 import { IWithResponsiveModeState } from 'office-ui-fabric-react/lib/utilities/decorators/withResponsiveMode';
 import { KeytipData } from '../../KeytipData';
 import { Panel, IPanelStyleProps, IPanelStyles } from '../../Panel';
-import { ResponsiveMode, withResponsiveMode } from 'office-ui-fabric-react/lib/utilities/decorators/withResponsiveMode';
+import { ResponsiveMode } from 'office-ui-fabric-react/lib/utilities/decorators/withResponsiveMode';
 import {
   SelectableOptionMenuItemType,
   getAllSelectedOptions,
   ISelectableDroppableTextProps,
 } from 'office-ui-fabric-react/lib/utilities/selectableOption/index';
 // import and use V7 Checkbox to ensure no breaking changes.
 import { Checkbox, ICheckboxStyleProps, ICheckboxStyles } from 'office-ui-fabric-react/lib/Checkbox';
+import { getPropsWithDefaults } from '@uifabric/utilities';
+import { useResponsiveMode } from 'office-ui-fabric-react/lib/utilities/hooks/useResponsiveMode';
+import { useMergedRefs, usePrevious } from '@uifabric/react-hooks';
 
 const getClassNames = classNamesFunction<IDropdownStyleProps, IDropdownStyles>();
 
 /** Internal only props interface to support mixing in responsive mode */
-export interface IDropdownInternalProps extends IDropdownProps, IWithResponsiveModeState {}
+export interface IDropdownInternalProps extends IDropdownProps, IWithResponsiveModeState {
+  hoisted: {
+    rootRef: React.Ref<HTMLDivElement>;
+    selectedIndices: number[];
+    setSelectedIndices: React.Dispatch<React.SetStateAction<number[]>>;
+  };
+}
 
 export interface IDropdownState {
```

Comment:
Should probably stop exporting this too if possible, since I don't think it's used by anything that needs to be exported.